_includes/manuals/3.15/1.2_release_notes.md

```diff
@@ -4,11 +4,129 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
+#### Upgrading made easier
+
+To get the right resources for the upgrade, you can now go to Administer &gt; Upgrade which contains helpful links.
+
+#### Easily remove oVirt resources
+
+With oVirt being deprecated, you may use a new rake task that will remove them for you.
```

Comment:
I don't think this is a headline feature and it's already mentioned in the deprecations